hc/settings.py

```diff
@@ -121,8 +121,12 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
-PING_ENDPOINT = SITE_ROOT + "/ping/"
+SITE_ROOT = os.environ.get('SITE_ROOT')
```

Comment:
os.environ takes a default/ fall-back value, as a second parameter (in case the variable is not set )  
`SITE_ROOT = os.environ.get('SITE_ROOT','http://localhost:8000')`
- This will reduce your reduce your code by 3 line and help us stick to DRY principle